code/transcription.py

```diff
@@ -0,0 +1,213 @@
+# CREATED: 2/17/16 3:09 PM by Justin Salamon <justin.salamon@nyu.edu>
+
+'''
+Compare transcription results reported in MIREX 2015 for the Note Tracking
+subtask of multi-f0 challenge against those produce by the mir_eval
+transcription module
+'''
+
+import numpy as np
+import mir_eval
+import glob, os
+import pandas as pd
+import sys
+
+
+def load_mirex_scores(alg):
+    '''
+    Load mirex scores for a specific algorithm
+
+    Parameters
+    ----------
+    alg : str
+        Algorithm name
+
+    Returns
+    -------
+    mirex_scores : np.ndarray, shape=(10,6)
+        The scores. There are 10 tracks, and 6 evaluation metrics: Precision,
+        Recall, F-measure, Precision_no_offset, Recall_no_offset,
+        F-measure_no_offset
+    '''
+
+    mirex_scores = []
+
+    mirex_results_path = \
+        'data/transcription/mirex2015_su/mirex_results/' \
+        '%s/%s.results.csv' % (alg, alg)
+    mirex_nooffset_results_path = \
+        'data/transcription/mirex2015_su/mirex_results/' \
+        '%s/%s.onsetOnly.results.csv' % (alg, alg)
+
+    mirex_results = pd.read_csv(mirex_results_path)
+    mirex_results = mirex_results.fillna(0)
+    precision = mirex_results['Precision'].tolist()[:10]
+    recall = mirex_results['Precision'].tolist()[:10]
+    f1 = mirex_results['Ave. F-measure'].tolist()[:10]
```

Comment:
Are there more than 10 scores to compare?  Once you nail down the differences more closely, it will be valuable to report these differences for all the scores you can get ahold of, because why not.